app/utils/FeatureSwitch.scala

```diff
@@ -32,7 +32,7 @@ object FeatureSwitch {
     val sysPropValue = sys.props.get(systemPropertyName(name))
     sysPropValue match {
       case Some(x) => x.toBoolean
-      case None => Play.configuration.getBoolean(confPropertyName(name)).getOrElse(false)
+      case None => appConfig.conf.getConfBool(confPropertyName(name), false)
```

Comment:
I think it's worth keeping the functionality consistent here, as above. `getConfBool` checks a different part of the configuration.